src/routers/loginController.ts

```diff
@@ -0,0 +1,20 @@
+import db from '../models'

```

Comment:
Controller or route is used to handle the rest api path. Then controller call the service to run the specified function. We are putting the function in wrong directory. They are supposed in service directory